test/component-bind.spec.ts

```diff
@@ -99,24 +99,6 @@ describe('component', () => {
         );
       });
 
-      it('throws on access', () => {
-        class FailedComponent extends Composite {
-
-          @bind('#textInput1.text')
-          myText: string;
-
-          constructor() {
-            super({});
-            this.append(new TextInput({id: 'textInput1'}));
-          }
-
-        }
-        expect(() => new FailedComponent().myText).to.throw(
-          'Binding "myText" <-> "#textInput1.text" failed to provide FailedComponent property "myText": '
-          + 'FailedComponent is not a @component'
```

Comment:
Because it would not throw on initialization (still doesn't, throws after a timeout) and therefore you can/could still access the property. Originally the property was in a non-functioning state at that point, so there had to be an error. Now it just works like a normal property, just without the binding.